extensions.json

```
@@ -1347,6 +1379,11 @@
       "repository": "https://github.com/zabel-xyz/plsql-language",
       "version": "1.8.2"
     },
+    {
+      "id": "yatki.vscode-surround",
+      "repository": "https://github.com/yatki/vscode-surround.git",
+      "version": "1.0.2"
```

Here we can checkout the right tag as well:

```suggestion
      "repository": "https://github.com/yatki/vscode-surround",
      "version": "1.0.2",
      "checkout": "v1.0.2"
```